move/axelar_gateway/sources/gateway.move

```diff
@@ -176,8 +190,8 @@ entry fun approve_messages(
     message_data: vector<u8>,
     proof_data: vector<u8>,
 ) {
-    let messages = peel_messages(*&message_data);
-    let proof = peel_proof(proof_data);
+    let proof = peel_data!(proof_data, |bcs| proof::peel(bcs));
```

Comment:
```suggestion
    let proof = peel_data!(proof_data, proof::peel);
```
does this work instead since it's already a method or does move macros require it to have the function form?

Reply:
i've tried; it doesn't work without lambda